_includes/navigation.html

```diff
@@ -1,8 +1,8 @@
 <nav class="navbar shadow-none fixed-top navbar-expand-lg" id="sectionsNav">
     <div class="container">
         <div class="navbar-translate">
-            <a class="navbar-brand" href="#">
-                <img src="{{ site.baseurl }}/assets/img/logo/logo.png">
+            <a target="_blank" class="navbar-brand" href="http://sack.edu.lk/">
+                <img src="{{ site.baseurl }}/assets/img/logo/nav-logo.svg">
```

Comment:
Let's open it within the same page.
```suggestion
            <a class="navbar-brand" href="http://sack.edu.lk/">
                <img src="{{ site.baseurl }}/assets/img/logo/nav-logo.svg">
```

Reply:
now @jayasanka-sack